### ISSUE TITLE
QOI support

### ISSUE THREAD

**Issue by Keterion** [deckerst/aves#1119]
**Is your feature request related to a problem? Please describe.**
No problems with the app, only was considering using qoi instead of png and there doesn't seem to be any support

**Describe the solution you'd like**
QOI images being recognized and supported by the app

**Describe alternatives you've considered**
Not using the format
Look for other apps, there don't seem tk be any though

**Additional context**

**Comment by deckerst** [comment]
Never heard of the format. Is it used a lot in a specific field, like TIFF in the scientific world?

In any case, I don't intend to support niche formats, even if the format is supposedly better than the ones in use. It's like JPEG XL (cf #730). The app relies on the image codecs provided by the device (except for TIFF).

**Comment by Keterion** [comment]
No, the format is pretty niche and not used in any specific field as far as I know.
I guess support for it shouldn't be expected then.

Still thank you for developing the app and the fast respose though

**Comment by NHodgesVFX** [comment]
QOI is interesting as it is similar to PNG in quality but is several times faster and can implemented in around 300 lines of code.

https://qoiformat.org/

There is also the QOA audio format as well.
https://qoaformat.org/
